model/ConsumerService.php

```diff
@@ -64,14 +66,35 @@ public function getConfiguration($iss = '')
             $res = $instance->getPropertiesValues([
                 self::PROPERTY_KEY,
                 self::PROPERTY_SECRET,
+                self::PROPERTY_ENTRY_POINT,
             ]);
 
             $config = [
                 self::PROPERTY_KEY => $res[self::PROPERTY_KEY][0]->literal,
                 self::PROPERTY_SECRET => $res[self::PROPERTY_SECRET][0]->literal,
+                self::PROPERTY_ENTRY_POINT => $res[self::PROPERTY_ENTRY_POINT][0]->literal,
             ];
         }
 
         return $config;
     }
+
+    /**
+     * @return array
+     */
+    public function getAvailableEntryPoints()
```

Comment:
I think this is something more generic than relations between NCCER and this extention, as it should be undependent and reusable. The purpose of this special entry points is to define custom code specific to each ISS, that have to  manage token according to valudation\parsing\whatever rules specific to each customer. As for now we are sharing the only one url with third party systems